src/AST-Core/RBLiteralValueNode.class.st

```diff
@@ -67,6 +67,12 @@ RBLiteralValueNode >> evaluateForReceiver: aReceicer [
 	^ value
 ]
 
+{ #category : #testing }
+RBLiteralValueNode >> isString [
```

Comment:
Having isString true for non String classes is a bad idea. It breaks the way to check that the object is a kind of string. I dont mean a kind of String class but the object implementing its protocol. I bet we have many users of isString assuming this semantics

Reply:
We could rename it as isValueString?

Reply:
or isLiteralStringNode?